Subject: [PATCH] update xmon helptext                                                                                                                                
From: Olaf Hering <olh@suse.de>                                                                                                                                      

xmon will display nothing but noise on a G5 if BOOTX_TEXT is not enabled.
mention the recognized kernel cmdline options for xmon.

Signed-off-by: Olaf Hering <olh@suse.de>

 arch/ppc64/Kconfig.debug |    9 +++++++++
 1 files changed, 9 insertions(+)

Index: linux-2.6.12/arch/ppc64/Kconfig.debug
===================================================================
--- linux-2.6.12.orig/arch/ppc64/Kconfig.debug
+++ linux-2.6.12/arch/ppc64/Kconfig.debug
@@ -41,10 +41,19 @@ config XMON
 	help
 	  Include in-kernel hooks for the xmon kernel monitor/debugger.
 	  Unless you are intending to debug the kernel, say N here.
+	  Make sure to enable also CONFIG_BOOTX_TEXT on Macs. Otherwise
+	  nothing will appear on the screen (xmon writes directly to the
+	  framebuffer memory).
+	  The cmdline option 'xmon' or 'xmon=early' will drop into xmon very
+	  early during boot. 'xmon=on' will just enable the xmon debugger hooks.
+	  'xmon=off' will disable the debugger hooks if CONFIG_XMON_DEFAULT is set.
 
 config XMON_DEFAULT
 	bool "Enable xmon by default"
 	depends on XMON
+	help
+	  xmon is normally disabled unless booted with 'xmon=on'.
+	  Use 'xmon=off' to disable xmon init during runtime.
 
 config PPCDBG
 	bool "Include PPCDBG realtime debugging"
